src/main/resources/application.properties

```diff
@@ -4,8 +4,20 @@ spring.datasource.url=jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}
 spring.datasource.username=${DB_USERNAME}
 spring.datasource.password=${DB_PASSWORD}
 
+#email config
+spring.mail.host=smtp.gmail.com
```

Comment:
I'd rather not have the email host and port exposed in here, it's best to have that in an env as well